feat(rust): add clould invitation subcommands

polvorin opened the pull request:
```   
   $ ockam invitation create|list|accept|reject
```
Notes:

* The CBOR type tags are removed to make structs compatible with elixir side, this is a temporary workaround.

<!-- Thank you for sending a pull request :heart: -->

## Current Behaviour

<!-- Please describe the current behavior of the code before the changes in this pull request is applied. -->

## Proposed Changes

<!-- Please describe the changes proposed in this pull request. -->
<!-- If this pull request resolves an already recorded bug or a feature request, please add a link to that issue. -->

## Checks

<!-- To help us review and merge this pull request quickly, please confirm the following:  -->

- [X] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).
- [X] All commits in this Pull Request follow the Ockam [commit message convention](https://www.ockam.io/learn/how-to-guides/contributing/CONTRIBUTING#commit-messages).
- [X] I accept the Ockam Community [Code of Conduct](https://www.ockam.io/learn/how-to-guides/high-performance-team/conduct/).
- [X] I have accepted the Ockam [Contributor License Agreement](https://www.ockam.io/learn/how-to-guides/contributing/cla/) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/contributors](https://github.com/build-trust/contributors) repository.

<!-- Looking forward to merging your contribution!! -->

polvorin committed:
feat(rust): add clould invitation subcommands

ockam invitation create|list|accept|reject

Notes:

* The CBOR type tags are removed to make structs compatible
with elixir side, this is a temporary workaround.

mrinalwadhwa commented:
> This introduces a top-level "cloud" subcommand, with only the "invitation" one as child

Adrian's impl of project and space where initially `cloud project` and `cloud space`. We moved them to the top level last week.

I think we should keep `invitation` at the top level as well. This ties better into the overall structure that is emering.

polvorin commented:
> > This introduces a top-level "cloud" subcommand, with only the "invitation" one as child
> 
> Adrian's impl of project and space where initially `cloud project` and `cloud space`. We moved them to the top level last week.
> 
> I think we should keep `invitation` at the top level as well. This ties better into the overall structure that is emering.

Removed the `cloud` subcommand
